packages/fxa-graphql-api/src/test/lib/oauth/index.spec.ts

```diff
@@ -0,0 +1,88 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Container } from 'typedi';
+import { Request, Response } from 'express';
+import { assert } from 'chai';
+import 'mocha';
+import sinon from 'sinon';
+
+import { loggerContainerKey, configContainerKey } from '../../../lib/constants';
+
+const sandbox = sinon.createSandbox();
+// tslint:disable-next-line: no-empty
+const mockLogger = { debug: () => {} };
+Container.set(loggerContainerKey, mockLogger);
+const mockConfig = {
+  get: (k: string): string | number => {
+    const fakeConfigs: { [key: string]: string | number } = {
+      authHeader: 'authorization',
+      'oauth.accessToken.hexLength': 64,
+    };
+    return fakeConfigs[k];
+  },
+};
+Container.set(configContainerKey, mockConfig);
+
+import { getTokenId, oauthBearerTokenValidator } from '../../../lib/oauth';
+import { AppConfig } from '../../../config';
+
+describe('oauthBearerTokenValidator', () => {
+  const sendStub = sandbox.stub();
+  const statusStub = sandbox.stub().returns({ send: sendStub });
+  const res = ({ status: statusStub } as unknown) as Response;
```

Comment:
I know why this is here, and its one of the few things in TypeScript that still annoys me. 😬